### PR TITLE
added hidden page

### PR DESCRIPTION
**Pull request by nielsmijnster**

### CODE REVIEW

File: html/hidden.html [hunk 1]
```diff
@@ -0,0 +1 @@
+<html style="height: 100%;"><head><meta name="viewport" content="width=device-width, minimum-scale=0.1"><title>aboutme.png (992×1058)</title></head><body style="margin: 0px; background: #0e0e0e; height: 100%"><img style="display: block;-webkit-user-select: none;margin: auto;cursor: zoom-out;background-color: hsl(0, 0%, 90%);transition: background-color 300ms;" src="https://courses.cs.washington.edu/courses/cse190m/12sp/labs/1/images/aboutme.png" width="992" height="1058"></body></html>
```
Comment (flaviostutz) [inline html/hidden.html]: Pls format this HTML code. You can use VSCode plugins for that

**Review by flaviostutz — CHANGES_REQUESTED** [review]
Pls format the code

File: html/hidden.html (hunk 1 above)
Comment (nielsmijnster) [reply html/hidden.html]: aah nice, yeah i just used prettier, thanks for the remark!

**Review by flaviostutz — APPROVED** [review]
Nice!

**Review by flaviostutz — APPROVED** [review]
Very nice, man!!!

**Pull request merged**